org.syphon.Syphon.json

```
@@ -0,0 +1,97 @@
+{
+    "app-id": "org.syphon.Syphon",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "41",
+    "sdk": "org.gnome.Sdk",
+    "command": "syphon",
+    "separate-locales": false,
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=fallback-x11",
+        "--socket=wayland",
```

```suggestion
        "--socket=fallback-x11",
        "--socket=wayland",
```

This assume it actually support wayland